Specs/Scene/ModelSpec.js

```diff
@@ -1041,6 +1042,41 @@ defineSuite([
         });
     });
 
+    it('loads a glTF 2.0 model with AGI_articulations extension', function() {
+        return loadModel(boxArticulationsUrl).then(function(m) {
+            verifyRender(m);
+
+            m.setArticulationStage('SampleArticulation MoveX', 1.0);
+            m.setArticulationStage('SampleArticulation MoveY', 2.0);
+            m.setArticulationStage('SampleArticulation MoveZ', 3.0);
+            m.setArticulationStage('SampleArticulation Yaw', 4.0);
+            m.setArticulationStage('SampleArticulation Pitch', 5.0);
+            m.setArticulationStage('SampleArticulation Roll', 6.0);
+            m.setArticulationStage('SampleArticulation Size', 0.9);
+            m.setArticulationStage('SampleArticulation SizeX', 0.8);
+            m.setArticulationStage('SampleArticulation SizeY', 0.7);
+            m.setArticulationStage('SampleArticulation SizeZ', 0.6);
+            m.applyArticulations();
+
+            var node = m.getNode('Root');
+            expect(node.useMatrix).toBe(true);
+
+            var expected = [
+                0.7147690483240505, -0.04340611926232735, -0.0749741046529782, 0,
+                -0.06188330295778636, 0.05906797312763484, -0.6241645867602773, 0,
+                0.03752515582279579, 0.5366347296529127, 0.04706410108373541, 0,
+                1, 3, -2, 1
+            ];
+
+            var matrix = node.matrix;
+            for (var i = 0; i < 16; ++i) {
```

Comment:
I'm pretty sure you can compare two arrays directly with `toEqualEpsilon`.

Reply:
Ah it [does](https://github.com/AnalyticalGraphicsInc/cesium/blob/3f34fed5975ba35274cd1b4d90ff671977821084/Specs/addDefaultMatchers.js#L151), I never knew that, thanks.